android/app/src/main/java/org/fossasia/openevent/activities/SplashActivity.java

```diff
@@ -1,32 +0,0 @@
-package org.fossasia.openevent.activities;
```

Comment:
Why did you remove the splash activity?
Even though it doesn't serves any meaningful purpose as of now, but we can use this for prefetching the session data and other time consuming tasks on app launch.